Cargo.toml

```
@@ -41,12 +40,16 @@ rustwide = "0.6.0"
 tempdir = "0.3"
 mime_guess = "2"
 
+# json {de}serialization
+serde_json = "1.0.51"
+serde = { version = "1.0.106", features = ["derive"] }
```

These exact version numbers are ignored by cargo. Please change them to the semver that cargo actually listens to:

```suggestion
serde_json = "1"
serde = { version = "1", features = ["derive"] }
```